### ISSUE TITLE
Github search might be broken.

### ISSUE THREAD

**Issue by akki199421** [isaacs/github#1446]
I was just looking at the most starred repos of all time and used the following command:

[https://github.com/search?o=desc&q=stars%3A%3E0&s=stars&type=Repositories](https://github.com/search?o=desc&q=stars%3A%3E0&s=stars&type=Repositories
)
This search produces inconsistent results, where a library may not be shown.

![screenshot 2018-12-28 at 6 17 20 pm](https://user-images.githubusercontent.com/7352401/50515893-67f20b00-0acd-11e9-8610-c8b15b0d850d.png)
![screenshot 2018-12-28 at 6 17 27 pm](https://user-images.githubusercontent.com/7352401/50515894-67f20b00-0acd-11e9-900b-6b764f910ed7.png)

Notice that the first image has both vue and react but the second one doesn't have vue.

I refreshed the pages several times and the issue persisted. Checked the console but there was no error.

Is it just me or did somebody else also noticed this as well?

**Comment by cirosantilli** [comment]
Yes, if you search >1 stars sort by stars shows twbs/bootstrap on top, >100 shows freeCodeCamp/freeCodeCamp (correct).

Likely >1 is overflowing something.

**Comment by markbernard** [comment]
Definitely still broken. I searched for all repositories with language Java (https://github.com/search?l=&o=desc&q=language%3AJava&s=stars&type=Repositories) and the top entry was 1.2k stars. There are definitely a bunch with more stars.
